packages/next/src/build/webpack-config.ts

```diff
@@ -1828,11 +1828,11 @@ export default async function getBaseWebpackConfig(
             appDir: appDir,
             pagesDir: pagesDir,
             esmExternals: config.experimental.esmExternals,
-            outputFileTracingRoot: config.experimental.outputFileTracingRoot,
+            outputFileTracingRoot: config.outputFileTracingRoot,
             appDirEnabled: hasAppDir,
             turbotrace: config.experimental.turbotrace,
             optOutBundlingPackages,
-            traceIgnores: config.experimental.outputFileTracingIgnores || [],
+            traceIgnores: [],
```

Comment:
```suggestion
            traceIgnores: config.outputFileTracingExcludes || [],
```

shouldn't we replace to `outputFileTracingExcludes`?

Reply:
Those are per-page on a glob level these were global and applied to all tracing

Reply:
Ah, got it!